README.md

```diff
@@ -111,6 +112,6 @@ state, make sure to strictly pin your requirements to `broadcaster==0.2.0`.
 To be more capable we'd really want to add some additional backends, provide API support for reading recent event history from persistent stores, and provide a serialization/deserialization API...
 
 * Serialization / deserialization to support broadcasting structured data.
-* Backends for Redis Streams, Apache Kafka, and RabbitMQ.
+* Backends for Apache Kafka, and RabbitMQ.
```

Comment:
🤔

Reply:
This is `What's next` section. Redis Streams lands with this PR.